docs/advanced-features/middleware.md

```diff
@@ -31,7 +31,7 @@ To begin using Middleware, follow the steps below:
 npm install next@latest
 ```
 
-2. Create a `middleware.ts` (or `.js`) file at the same level as your `pages` directory
+2. Create a `middleware.ts` (or `.js`) file at root directory
```

Comment:
At the same level is correct as it can go in the src directory as well if present. Maybe we can word this differently as we've had a few PRs try to change this to root but root isn't accurate 🤔